modules/acm/src/main/java/edu/gemini/epics/acm/CaSimpleObserveSenderImpl.java

```diff
@@ -0,0 +1,775 @@
+/*
+ * Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+ * For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+package edu.gemini.epics.acm;
+
+import edu.gemini.epics.EpicsReader;
+import edu.gemini.epics.EpicsService;
+import edu.gemini.epics.ReadOnlyClientEpicsChannel;
+import edu.gemini.epics.api.ChannelListener;
+import edu.gemini.epics.impl.EpicsReaderImpl;
+import gov.aps.jca.CAException;
+import gov.aps.jca.TimeoutException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+/*
+ * Class to handle observe commands in instruments were there is no CAR for the apply. In those cases, The observe CAR
+ * handles all the command state.
+ */
+public class CaSimpleObserveSenderImpl<C extends Enum<C> & CarStateGeneric> implements ApplySenderWithResource {
```

Comment:
Will this be used only for GSAOI?

Reply:
Yes. It could be used for other instruments thought (NIFS comes to mind)